cloudformation.yaml

```diff
@@ -116,7 +116,7 @@ Resources:
                 - !Sub arn:aws:autoscaling:${AWS::Region}:${AWS::AccountId}:autoScalingGroup:*:autoScalingGroupName/*
                 Condition:
                   StringLike:
-                    autoscaling:ResourceTag/RotateWithElasticsearchNodeRotation: "*"
+                    autoscaling:ResourceTag/RotateWithElasticsearchNodeRotation: "true"
```

Comment:
Just checked, we do indeed have the tag value set to `true`:

https://eu-west-1.console.aws.amazon.com/ec2autoscaling/home?region=eu-west-1#/details/Ophan-ES7-WarmDataAutoscalingGroup-1X4AC7UAK7UAN?view=details

![image](https://user-images.githubusercontent.com/52038/151013604-4d161e31-5008-4afa-9f48-d54e53e4b555.png)